### PR TITLE
remvoed the colourful table and changed the table for details 

### PR DESCRIPTION
**Pull request by naderakhgari**
…izentally ot vertically

Thank you for contributing! To help the review process, please provide the following:

### Proposal
remvoed the colourful table and changed the table for details form horizontally to vertically 

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/Components/Results.js [hunk 1]
```diff
@@ -210,16 +210,18 @@ export default function Results(props) {
...
 									className="quiz-button card-button"
 									onClick={nextAttempt}
 									disabled={attemptNumber <= 1}
 								>
-                Next Attempt
+                  Next Attempt
 								</button>
-								<span>All Attempt : {attemptCounter - attemptNumber+1}</span>
+								<span>
+                  Current Attempt : {attemptCounter - attemptNumber + 1}
```
Comment (pendletons) [inline client/src/Components/Results.js:223]: 👍

File: client/src/Components/Results.js [hunk 2]
```diff
@@ -288,7 +293,7 @@ export default function Results(props) {
 																		className="quiz-button"
 																		onClick={() => showDetail(student)}
 																	>
-                                    Show Ditails
+                                      Show Ditails
```
Comment (pendletons) [inline client/src/Components/Results.js:296]:
```suggestion
                                      Show Details
```

**Review by pendletons — COMMENTED** [review]
Can you add a screenshot of how this looks?

**Review by naderakhgari — COMMENTED** [review]
fixed

**Comment by cockochan** [comment]
What do you mean "removed"? Show the screenshot

File: client/src/Components/Results.js (hunk 2 above)
Comment (naderakhgari) [reply client/src/Components/Results.js]: image

I changed the details table for one student to be vertically
we have the score table
the big colourful table is removed

Comment (naderakhgari) [reply client/src/Components/Results.js]: I can not access to the exact view of small table 
it is showing just vertically

Comment (naderakhgari) [reply client/src/Components/Results.js]: would you prefer to have the big colourful table ?

**Review by pendletons — APPROVED** [review]

**Pull request merged**